.docker/Dockerfile

```diff
@@ -0,0 +1,58 @@
+FROM composer/composer:latest-bin AS composer
+
+FROM ghcr.io/shyim/wolfi-php/frankenphp:8.3 AS base
+ARG UID=10001
```

Comment:
```
❯ id
uid=501(xepozz) gid=20(staff)
```
On my Macbook, just in case

Reply:
Yes. These are set to your IDs via make.